docs/DeveloperGuide.md

```diff
@@ -182,33 +186,65 @@ given for dates
+* `Meeting`   - its `From`'s `LocalDateTime` value in ascending chronological order 
+* `Deliverable`  - its `Deadline`'s `LocalDateTime` value in ascending chronological order 
+* `Person`   - its `Title`'s `String` value in ascending alphabetical order 
 
-Autosort is faciliated by custom objects that implements `Comparator`.
+Auto-sort is facilitated by custom classes that implements `Comparator`.
 
-The following sequence diagram shows how a list is autosorted upon an addition of a new element.
+The following sequence diagram shows how a list is auto-sorted upon an addition of a `Meeting`.
 
-![UndoSequenceDiagram](images/AutosortSequenceDiagram.png)
+![AutosortSequenceDiagram](images/AutosortSequenceDiagram.png)
 
 #### Design consideration:
 
-##### Aspect: How autosorting executes
+##### Aspect: How auto-sorting executes
 
 * **Alternative 1 (current choice):** Sorts a list upon an addition or update of an element.
     * Pros: Error-free and easy to implement.
     * Cons: Relatively high time complexity i.e. O(nlogn).
 * **Alternative 2:** Searches the correct index in the list to insert an element upon addition or update.
     * Pros: Relatively low time complexity i.e. O(logn).
     * Cons: Prone to error and difficult to implement.
+    
+### Calendar feature
+
+#### Implementation
+
+The Calendar feature allows users to view their `Deliverable`s and`Meeting`s together in one chronologically ordered list - `calendarList`. 
+Specifically, the Calendar feature combines and orders `Deliverable`s and `Meeting`s by the following attributes: 
+
+* `Meeting`   - its `From`'s `LocalDateTime` value 
+* `Deliverable`  - its `Deadline`'s `LocalDateTime` value
+
+The combining is done by applying polymorphism; `Deliverable` and `Meeting` implement the interface `TimeEvent`.
+The following class diagram demonstrates the above-mentioned implementation. 
+![TimeEventClassDiagram](images/TimeEventClassDiagram.png)
+
+Meanwhile, the ordering is facilitated by the [Auto-sort feature](#auto-sort-feature).
+
+The following sequence diagram shows how the Calendar is updated upon an addition of a `Deliverable`.
+
+![CalendarSequenceDiagram](images/CalendarSequenceDiagram.png)
+
+#### Design consideration:
+
+##### Aspect: How the `calendarList` in `Calendar` is updated 
+
+* **Alternative 1 (current choice):** Holds references to `internalList`s in `UniqueDeliverableList` and `UniqueMeetingList`.
+    * Pros: Coupling is reduced as the implementation of `UniqueDeliverableList` and `UniqueMeetingList` are unmodified. 
+    * Cons: Relatively high time complexity as it handles all elements in a one-element update.
+* **Alternative 2:** Modifies implementation of `UniqueDeliverableList` and `UniqueMeetingList`. 
+    * Pros: Relatively low time complexity as for every one-element update to the `internalList` in `UniqueDeliverableList` or `UniqueMeetingList`, 
+        the same amount is updated in the `calendarList` in Calendar. 
```

Comment:
not very sure what this means

Reply:
hmmmm, i meant to say lower time complexity than adding back all meetings and deliverables, then sort them again for every update

Reply:
abit vague, not sure why "Modifying implementation of `UniqueDeliverableList` and `UniqueMeetingList`" would lower time complexity